src/kernel-install/ukify.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
```

Comment:
```suggestion
#!/usr/bin/python3
```
Not sure, but the package `python-is-python3` is still a thing in Debian, so better safe than sorry.